pkg/cloudprovider/fake/errors.go

```diff
@@ -0,0 +1,32 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+// fakeError implements controllers.RetryableError & controllers.CodedError
+type fakeError struct {
+	error
+	retryAble bool
```

Comment:
I was gonna suggest `retriable`, but it looks like this is a fairly [deep rabbit hole](https://english.stackexchange.com/questions/305273/retriable-or-retryable). I think retryable is actually correct -- go figured. Either way, A(ble) shouldn't be capitalized.